lib/FunctionIsolation/EnforceABI.cpp

```diff
@@ -114,7 +114,7 @@ static Register X("enforce-abi", "Enforce ABI Pass", true, true);
 static Logger<> EnforceABILog("enforce-abi");
 
 bool EnforceABI::prologue() {
-  FunctionDispatcher = M.getFunction("function_dispatcher");
+  FunctionDispatcher = getRegisteredIRHelper("function_dispatcher", M);
```

Comment:
Nice work. Eventually we might have a global variable for each helper that also describes its prototype.

But this is a good intermediate step.

Reply:
If you want that's easy to do even now - just pass it into the constructor of the registry and let it store it.

Reply:
Next time, let's focus on finishing this PR.